docs/how-to/index.md

```diff
@@ -10,4 +10,5 @@ deploy_sdcore_cups
 deploy_sdcore_gnbsim
 integrate_sdcore_with_observability
 enable_hugepages_for_upf
+backup_sdcore
 ```
```

Comment:
It looks like we're missing the `restore` document

Reply:
Good catch, fixed